swarm_copy/app/database/sql_schemas.py

```diff
@@ -0,0 +1,63 @@
+"""Schemas for the chatbot."""
+
+import datetime
+import uuid
+
+from sqlalchemy import DateTime, ForeignKey, Integer, String
+from sqlalchemy.orm import DeclarativeBase, Mapped, mapped_column, relationship
+
+
+class Base(DeclarativeBase):
+    """Base declarative base for SQLAlchemy."""
+
+    pass
+
+
+def uuid_to_str() -> str:
+    """Turn a uuid into a string."""
+    return uuid.uuid4().hex
+
+
+def utc_now() -> datetime.datetime:
+    """Return the utc time."""
+    return datetime.datetime.now(datetime.timezone.utc).replace(tzinfo=None)
+
+
+class Threads(Base):
+    """SQL table for the users thread / conversations."""
+
+    __tablename__ = "threads"
+    thread_id: Mapped[str] = mapped_column(
+        String, primary_key=True, default=uuid_to_str
+    )
+    vlab_id: Mapped[str] = mapped_column(
+        String, default="430108e9-a81d-4b13-b7b6-afca00195908"
```

Comment:
This default value as well as the project_id seem like 'magic constants' (https://en.wikipedia.org/wiki/Magic_number_(programming)).
What I mean is if someone new looks at the code they will not know why these particular ids were chosen as defaults. I would put them into named python constants where the name itself explains what the defaults are.

Reply:
This is only temporary and will be removed once we have the CURDs. Same for the `thread_id` that stays the same for now.

Reply:
I don't know how `mapped_column` works and which defaults it has, but if it is not yet the case I suggest `vlab_id` and `project_id` to be `nullable=False`.